src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx

```diff
@@ -31,6 +36,27 @@ const NERInput = styled(TextField)(({ theme }) => ({
 const CreateProjectFormView: React.FC<CreateProjectFormViewProps> = ({ states, allowSubmit, onCancel, onSubmit }) => {
   const { name, carNumber, crId, summary } = states;
   const theme = useTheme();
+  const [team, setTeam] = useState<Team | null>(null);
+  const { isLoading, data: teams } = useAllTeams();
+  if (isLoading || !teams) return <LoadingIndicator />;
+
+  const teamsSearchOnChange = (
+    _event: React.SyntheticEvent<Element, Event>,
+    value: { label: string; id: string } | null
+  ) => {
+    if (value) {
+      const team = teams.find((team: Team) => team.teamName === value.id);
+      if (team) {
```

Comment:
I believe the reason why it doesn’t keep the name after you select it is because you are checking the team.teamName versus the value.id, where value.id = the value.teamId, not the teamName. so its two different values that will never be equivalent, so it won’t save.